GitUI/GitUI.csproj

```
@@ -26,6 +26,7 @@
     <PackageReference Include="AppInsights.WindowsDesktop" Version="$(AppInsightsWindowsDesktopVersion)" />
     <PackageReference Include="Microsoft-WindowsAPICodePack-Core" Version="1.1.3.3" />
     <PackageReference Include="Microsoft-WindowsAPICodePack-Shell" Version="1.1.3.3" />
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
```

Shared references are configured via Solution.Versions.props
```suggestion
    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
```